pkg/controller/disruption/disruption_test.go

```diff
@@ -1025,3 +1038,64 @@ func TestDeploymentFinderFunction(t *testing.T) {
 		})
 	}
 }
+
+func TestUpdatePDBStatusRetries(t *testing.T) {
```

Comment:
This test verifies that it retries on a conflict error, but nowhere is it made clear in the code or comments why this is important. Can you mention the race here and that this is just the simplest way of avoiding it?

Reply:
I've drastically revamped the test so that it asserts on the underlying bad behavior rather than how we fix it. PTAL